beacon-chain/blockchain/receive_attestation.go

```diff
@@ -35,9 +35,8 @@ func (s *Service) ReceiveAttestationNoPubsub(ctx context.Context, att *ethpb.Att
 	ctx, span := trace.StartSpan(ctx, "beacon-chain.blockchain.ReceiveAttestationNoPubsub")
 	defer span.End()
 
-	_, err := s.onAttestation(ctx, att)
-	if err != nil {
-		return errors.Wrap(err, "could not process attestation")
+	if err := s.onAttestation(ctx, att); err != nil {
+		errors.Wrap(err, "could not process attestation")
```

Comment:
?

Reply:
Missing return

Reply:
Oh man. Copy pasta failed. Surprising Goland didnt alert me for that...

Reply:
Thankfully we have static analysis!